packages/components/src/components/PipelineRuns/PipelineRuns.scss

```diff
@@ -43,3 +43,7 @@ limitations under the License.
     }
   }
 }
+
+.bx--overflow-menu--flip.bx--overflow-menu-options::after{
```

Comment:
to avoid leaking styles to other elements on the consuming page:
```diff
-.bx--overflow-menu--flip.bx--overflow-menu-options::after {
+.definition .bx--overflow-menu--flip.bx--overflow-menu-options::after {
```

Reply:
ignore this, due to the way Carbon creates the menu we can't scope the style to its container, it's added as a direct child of `body`